app/grandchallenge/profiles/models.py

```diff
@@ -244,6 +245,41 @@ def dispatch_unread_direct_messages_email(
             subscription_type=EmailSubscriptionTypes.NOTIFICATION,
         )
 
+    @cached_property
+    def file_civs_user_has_permission_to_use(self):
```

Comment:
Is the users profile the logical place for this? It's only used by `InterfaceFormField`, and only depends on attributes stored on those instances, so could be a `cached_property` there, right?

Reply:
That is a fair point. I thought of it as a property of the user, but it is indeed only used on the InterfaceFormField, and I also don't foresee it being used anywhere else, so putting it there makes sense.